jira_completion.py

```diff
@@ -0,0 +1,138 @@
+"""
+Sublime Text 3 Plugin to autocomplete JIRA issues numbers
+by ticket titles.
+
+How to use:
+
+Type "JI" with an optional filter, e.g. "JIauthgoogle", press Alt+/.
+Completion menu should pop up.
+
+How to set up:
+1. Add your username and password to ~/.netrc, e.g.:
+
+    machine jira.iponweb.net login vpetrov password topsecretpassword
+
+CAUTION: async requests are not supported, so your editor will freeze
+until reply from JIRA is received.
+Also, minimal error handling, sorry.
+"""
+
+import base64
+import datetime
+import json
+import netrc
+import sublime
+import sublime_plugin
+import urllib
+
+from urllib.error import HTTPError
+
+
+class Settings:
+    def __init__(self, settings):
+        self._settings = settings
+
+    @property
+    def hostname(self):
+        return self._settings.get('hostname', 'jira.iponweb.net')
+
+    @property
+    def trigger(self):
+        return self._settings.get('trigger', 'JI')
+
+    @property
+    def jira_completion_url(self):
+        return self._settings.get(
+            'jira_completion_url',
+            'https://jira.iponweb.net/rest/api/2/search?'
+            'jql=(watcher+in+({username})+'
+                'OR+assignee+in+({username})+'
```

Comment:
May be use "was" instead of "is"? This will also select issues where assignee/reporter WAS anytime in past equal to username.